### PR TITLE
cockatrice: 2.8.0 -> 2.8.1-beta

### PR DESCRIPTION
**Pull request by kevung**
###### Motivation for this change
New version of Cockatrice

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Launched Cockatrice after local install with nix-env -f . -iA cockatrice in a clone of nixpkgs
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 116100` at fb3f9b87 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li><a href="https://gist.github.com/c1c2345025dae28a3391cc527adda7fe">cockatrice</a></li></li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md">unclear-gpl</a>

`gpl2` is a deprecated license, check if project uses `gpl2Plus` or `gpl2Only` and change `meta.license` accordingly.

Near pkgs/games/cockatrice/default.nix:25:5:
```
   |
25 |     license = lib.licenses.gpl2;
   |     ^
```
</li>
  </ul>
</details>

File: pkgs/games/cockatrice/default.nix [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 mkDerivation rec {
   pname = "cockatrice";
-  version = "2021-01-26-Release-2.8.0";
+  version = "2021-02-03-Development-2.8.1-beta";
 
   src = fetchFromGitHub {
     owner = "Cockatrice";
```
Comment (SuperSandro2000) [inline pkgs/games/cockatrice/default.nix:10]: You did not update the hash.

Comment (kevung) [reply pkgs/games/cockatrice/default.nix:10]: I am absolutely sorry... This should be fixed now. Thank you.

**Review by rmcgibbo — APPROVED** [review]
Looks like gpl2Plus https://github.com/Cockatrice/Cockatrice/blob/48c64587661fd8e9b79ecae3bac2615058804427/servatrice/src/main.cpp#L8

It'd be good to update the license to be more precise.

App builds and boots fine.
image

**Comment by kevung** [comment]
> Looks like gpl2Plus https://github.com/Cockatrice/Cockatrice/blob/48c64587661fd8e9b79ecae3bac2615058804427/servatrice/src/main.cpp#L8
> 
> It'd be good to update the license to be more precise.
> 
> App builds and boots fine.
> image

Thank you @rmcgibbo . I have updated the license into gpl2Plus.

**Pull request merged**